repository/src/test/java/org/alfresco/repo/version/ContentServiceImplTest.java

```diff
@@ -173,7 +174,7 @@ public void testWhenRequestContentDirectUrlIsNotSupported()
 
         try
         {
-            assertNull(contentService.requestContentDirectUrl(null, true, null));
+            assertNull(contentService.requestContentDirectUrl(null, null, true, null));
```

Comment:
Add -ve test for what happens if qname is passed in as "null" (and all other args are valid including nodeRef).

Reply:
This should be resolved now but please double check